cooperator/__init__.py

```diff
@@ -1,3 +1,13 @@
+from odoo import api, SUPERUSER_ID
+
 from . import models
 from . import report
 from . import wizard
+
+
+def post_init(cr, registry):
+    # the subscription journal must be created for each company.
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    subscription_request_model = env["subscription.request"]
+    for company in env.companies:
```

Comment:
no, indeed, i’ve just checked and it can be different (it depends on the `company_ids` field of the superuser, which can contain any company, not necessary all of them). i’ve fixed it while converting this post-init hook to a function call from the xml data file.